resources/visualizations/Top-Destination-Ports-By-Unique-Application-Count.json

```diff
@@ -0,0 +1,9 @@
+{
+   "title": "Top Destination Ports by Unique Application Count",
```

Comment:
This file should be called ```Top-Destination-Ports-By-Application-(histogram).json``` and the ```"title"``` should be ```"Top Destination Ports By Application (histogram)"``` (notice the capital B in ```By```, the extra dash between ```Application``` and ```(histogram)```)